api/Navigator.json

```diff
@@ -2240,7 +2245,11 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/vibrate",
           "support": {
             "chrome": {
-              "version_added": "32"
+              "version_added": "32",
+              "notes": [
```

Comment:
These notes aren't from mdn-bcd-collector, can you back them out and send a separate PR explaining them?